src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralFor.java

```diff
@@ -55,7 +55,7 @@ void testFor() {
         }
 
         //Invalid
-        for(int i = 0; i < 5; i++) {      // violation
+        for(int i = 0; i < 5; i++) {      // violation ''for' is not followed by whitespace'
```

Comment:
Not related to the changes you introduced but please remove the whitespace after `{`. A single space after `{` for violation message is sufficient :)

Reply:
done

Reply:
Not quite done, I assume that you have made changes on local, please push them to remote.

Reply:
ok i am sorry about that i have some misunderstanding

Reply:
done

Reply:
Resolved.